VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -1185,10 +1204,17 @@ subroutine vom_hourly_init ()
      &           + i_hd) * jmax25t_d(:)) / ((-1.d0 + p_E ** ((i_hd     &
      &           * (273.d0 + tair_h(th_) - topt_)) / (tair_h(th_)      &
      &           + 273.d0 * p_R_ * topt_))) * i_ha + i_hd)
-!     * (3.24), (Out[312])
-      rlt_h(:) = ((ca_h(th_) - gammastar) * o_pct * jmaxt_h(:)         &
+
+!       * fraction of absorbed radiation per crown area (Beer-lambert)
+        Ma_lt(:) = 1.0d0 - p_E ** (-lai_lt(:)/2.0d0)
+
```

Comment:
Please reflect in the definition of Ma_lt that it is actually per crown area. Don't you need to include cases for the old behaviour and the new behaviour including LAI?

Reply:
There are already cases, but I guess you don's see that anymore as it is already in your master by https://github.com/schymans/VOM/pull/26